src/cli_agent_orchestrator/clients/database.py

```
@@ -190,6 +218,80 @@ def _migrate_memory_indexes() -> None:
+        with sqlite3.connect(str(DATABASE_FILE)) as conn:
+            cursor = conn.execute("PRAGMA table_info(memory_metadata)")
+            columns = {row[1] for row in cursor.fetchall()}
+            if "access_count" not in columns:
+                conn.execute(
+                    "ALTER TABLE memory_metadata ADD COLUMN access_count INTEGER NOT NULL DEFAULT 0"
+                )
+                logger.info("Migration: added access_count column to memory_metadata")
+            if "last_accessed_at" not in columns:
+                conn.execute("ALTER TABLE memory_metadata ADD COLUMN last_accessed_at DATETIME")
+                logger.info("Migration: added last_accessed_at column to memory_metadata")
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)